.github/workflows/release.yml

```diff
@@ -57,13 +57,24 @@ jobs:
         env:
           CI: true
 
+      - name: Storing release version for changeset
+        id: release_version
+        run: echo "VALUE=$(./scripts/print_release_version.sh)" >> $GITHUB_OUTPUT
+        shell: bash
+        env:
+          GITHUB_TOKEN: ${{ secrets.RELEASE_GITHUB_TOKEN }}
```

Comment:
I was wondering what do we need this env for? 🤔

Reply:
It's needed for changeset (we execute `changeset version` in the script) as far as I know.